resources/styles/components/task/index.less

```diff
@@ -0,0 +1,35 @@
+.fixed-bar() {
+  left: 0;
+  position: fixed;
+  width: 100%;
+  background-color: fadeout(@tutor-white, 20%);
+}
+
+
+.task-view {
```

Comment:
Can you include `body.` to this to clarify that this is a class name on the root (worries me a little that this is outside of react)